contrib/builder/deb/aarch64/ubuntu-trusty/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/deb/aarch64/generate.sh"!
+#
+
+FROM aarch64/ubuntu:trusty
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libsqlite3-dev pkg-config libsystemd-journal-dev golang-1.6-go --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+RUN update-alternatives --install /usr/bin/go go /usr/lib/go-1.6/bin/go 100
+
+# Install Go
+# aarch64 doesn't have official go binaries, so use the version of go installed from
+# the image to build go from source.
+ENV GO_VERSION 1.7.3
+RUN mkdir /usr/src/go && curl -fsSL https://storage.googleapis.com/golang/go${GO_VERSION}.src.tar.gz | tar -v -C /usr/src/go -xz --strip-components=1 \
+	&& cd /usr/src/go/src \
+	&& GOOS=linux GOARCH=arm64 GOROOT_BOOTSTRAP="$(go env GOROOT)" ./make.bash
+
+ENV PATH /usr/src/go/bin:$PATH
+
```

Comment:
@hqhq I couldn't build it [and compared](https://github.com/docker/docker/blob/c5ceb0f945d390500a0045b14ffe4dcecfe46a85/contrib/builder/deb/armhf/ubuntu-trusty/Dockerfile#L9) the `contrib/builder/deb/aarch64/*/Dockerfile` with ` contrib/builder/deb/amrhf/*/Dockerfile`.
After adding

```Dockerfile
ENV AUTO_GOPATH 1
```

to all `contrib/builder/deb/aarch64/*/Dockerfile` I'm able to build the `docker-engine_*_arm64.deb` packages .